long_range_colloquium.md

```diff
@@ -0,0 +1,36 @@
+# Long Range Colloquium
+
+We aim to provide a platform for talks about latest developments in the fields of condensed matter and quantum information.
```

Comment:
I suggest being more assertive in this first sentence: we present a series of seminars about the latest developments in...
At this point we don't aim to provide a platform, we are using it :)
I'm new to GitHub, so let me know if I should have done something different from leaving a comment.

Reply:
All fixed, and this is exactly the right way to go.